deploy/olm-catalog/knative-eventing-operator/0.5.0/knative-eventing-operator.v0.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,609 @@
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - sources.eventing.knative.dev
+          resources:
+          - camelsources/status
+          - camelsources/finalizers
+          verbs:
+          - get
+          - update
+          - patch
+        - apiGroups:
+          - serving.knative.dev
+          resources:
+          - services
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - eventing.knative.dev
+          resources:
+          - channels
+          - brokers
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - camel.apache.org
+          resources:
+          - '*'
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - security.openshift.io
+          resourceNames:
+          - privileged
+          - anyuid
+          resources:
+          - securitycontextconstraints
+          verbs:
+          - use
+        - apiGroups:
+          - extensions
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - build.knative.dev
+          resources:
+          - '*/finalizers'
+          verbs:
+          - update
+        - apiGroups:
+          - policy
+          resources:
+          - podsecuritypolicies
+          verbs:
+          - create
+          - update
+          - patch
+          - get
+        - apiGroups:
+          - build.knative.dev
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          verbs:
+          - get
+          - create
+          - update
+          - delete
+        serviceAccountName: knative-eventing-operator
+      deployments:
+      - name: knative-eventing-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: knative-eventing-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: knative-eventing-operator
+            spec:
+              containers:
+              - command:
+                - knative-eventing-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: knative-eventing-operator
+                image: quay.io/openshift-knative/knative-eventing-operator:v0.5.0
+                args: ["--olm", "--install"]
```

Comment:
@jcrossley3 should I add fileame ?   I'd need to files ... one for "eventing core" and one for "sources" ?

Reply:
Yes, and you can comma-delimit them